lib/queuePopulator/LogReader.js

```diff
@@ -39,8 +39,7 @@ class LogReader {
         this.zkClient = params.zkClient;
         this.kafkaConfig = params.kafkaConfig;
         this.logConsumer = params.logConsumer;
-        this.pathToLogOffset = `${config.queuePopulator.zookeeperPath}/` +
-            `logState/${params.logId}/logOffset`;
+        this.pathToLogOffset = `/logState/${params.logId}/logOffset`;
```

Comment:
> is config.queuePopulator.zookeeperPath used elsewhere ? can't we just set it to an empty string in case of S3C ?

We cannot just set it as empty string as it used in the QueuePopulator path as the base path for zookeeper.

> if pathToLogOffset affects only Metadata log reader, should it not be moved to the dedicated LogReader subclass?

There's only one logReader class actually, the LogConsumer is what's custom per backend. This class however does not deal with offsets at all, it only handles consumption of the logs.

> this will affect the (deprecated) mongodb "replicaset" LogReader → not an issue since it is unused, but may be the occasion to remove it (here or in a separate ticket to be created)

This could be done in BB-481, as the Kafka implementation currently creates error logs when trying to store the offset.